### ISSUE TITLE
Support for RISE slides?

### ISSUE THREAD

**Issue by jlperla** [QuantEcon/sphinxcontrib-jupyter#22]
Hi there,
Is there any way to generate https://github.com/damianavila/RISE based slides at this point?  If not, it would be a great feature for generating slides for tutorials and classes.

**Comment by mmcky** [comment]
Hi @jlperla. I will add this idea to the list of future development features. It would be nice to target slides.

**Comment by nthiery** [comment]
That would be great indeed!
And in general having some syntax for setting cell metadata. Asecond use case: writing exercise sheets with automatic feedback / grading with [ordo](https://github.com/nbgallery/ordo) / [nbgrader](https://github.com/jupyter/nbgrader) respectively.

**Comment by mmcky** [comment]
I agree. We would like to add a specific ``jupyter`` directive for better control and more direct linking with the jupyter notebook spec. Ref #13

**Comment by mmcky** [comment]
Thanks @FelipeMaldonado -- PR https://github.com/QuantEcon/sphinxcontrib-jupyter/pull/112 adds support for `slides`. It is not feature complete - so will leave this issue open and tag for `0.6` for review.
